Podfile.lock

```diff
@@ -1,83 +1,101 @@
 PODS:
-  - Firebase/Analytics (7.5.0):
+  - Firebase/Analytics (8.3.0):
```

Comment:
same as in other PR - updates (especially such big ones like 8.x -> 7.x) should not be performed without a good reason or a task for it